app/views/find_page.py

```diff
@@ -14,17 +14,33 @@
 @find_page_bp.route("/find-page/invite")
 @confirmed_required
 def find_page_invite():
-    date_request_data = suggest_matches(db.session, current_user.id, 1)
-    if not date_request_data:
-        return render_template("main/find_page.html", date_request_data=None)
-
     date_request_form = DateRequestForm()
     is_requesting = True
-    date_request_data = date_request_data[0].profile
+    
+    date_request_data = get_suggested_matches_data(10)
+    if not date_request_data:
+        date_request_data=None
+        
     return render_template(
         "main/find_page.html", date_request_data=date_request_data, form=date_request_form, is_requesting=is_requesting
     )
 
+@find_page_bp.route("/find-page/refill", methods=["POST"])
+@confirmed_required
+def refill():
+    ignore_ids = [user["id"] for user in request.json]
+    return get_suggested_matches_data(10, ignore_ids=ignore_ids)
+
+def get_suggested_matches_data(limit : int, ignore_ids=None):
+    suggested_users = suggest_matches(db.session, current_user.id, limit, ignore_ids)
+    result = []
+    for user in suggested_users:
+        result.append({
+            "id" : user.id,
+            "name" : user.profile.name,
+            "description" : user.profile.description
```

Comment:
shouldn't we also include image here

Reply:
Yeah we should I left that out beceause there are no images in the project yet.